mungegithub/submit-queue/deployment/contrib/configmap.yaml

```diff
@@ -3,41 +3,42 @@ kind: ConfigMap
 metadata:
   name: contrib-sq-config
 data:
-  # basic config options.
-  config.http-cache-dir: /cache/httpcache
-  config.organization: kubernetes
-  config.project: contrib
-  config.pr-mungers: blunderbuss,lgtm-after-commit,submit-queue,check-labels
-  config.state: open
-  config.token-file: /etc/secret-volume/token
-  gitrepo.repo-dir: /gitrepos
-  gitrepo.enable-md-yaml: "false"
+  munge-config: |
+    # basic config options.
+    http-cache-dir: /cache/httpcache
```

Comment:
Agreed. We don't want someone changing one without changing the other. I'll make this change.